.travis.yml

```diff
@@ -19,8 +19,7 @@ before_install:
 install: travis_retry composer update --prefer-dist
 
 script:
-  - vendor/bin/phpcs
-  - vendor/bin/phpcs $(find tests/input/* | sort) --report=summary --report-file=phpcs.log; diff tests/expected_report.txt phpcs.log
+  - composer tests
```

Comment:
I would use `test` instead of `tests`, as it seems more natural.

I've seen it in [other places](https://github.com/thephpleague/skeleton/blob/master/composer.json#L37) too.